zkfacade/src/main/java/com/yahoo/vespa/curator/api/VespaCurator.java

```diff
@@ -0,0 +1,19 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.curator.api;
+
+import com.yahoo.path.Path;
```

Comment:
Could we use java.nio.file.Path instead? com.yahoo.path.Path is marked Beta and if we can use com.yahoo.path.Path internally only it would be nice. I haven't looked into if it's feasible, though

Reply:
`java.nio.file.Path` requires a `FileSystem` though, so not sure its suitable. `com.yahoo.path.Path` is better than `String` though.

Reply:
You are right, doesn't look like that will work. Agreed that it's better than `String`.